extensions/amp-timeago/validator-amp-timeago.protoascii

```diff
@@ -41,13 +41,6 @@ tags: {  # <amp-timeago>
   }
   attrs: { name: "locale" }
   attr_lists: "extended-amp-global"
-  cdata: {
-    # Regex: \s*[\w\p{L}\p{N}_.,:/+-]+(\s*[\w\p{L}\p{N}_.,:/+-]+)*\s*
-    # CDATA such as: Saturday 11 April 2017 00.37
-    # Note: Where Unicode characters may exist and a named group such as \w is
-    # used then it's necessary to also include \p{L}\p{N}_.
-    cdata_regex: "\\s*[\\w\\p{L}\\p{N}_\\.,:/+-]+(\\s*[\\w\\p{L}\\p{N}_\\.,:/+-]+)*\\s*"
```

Comment:
I think the original intention was to force some text to be here (it's used in case JS fails to properly "timeago" it). Can we do something like "there exists non-whitespace text"?

Reply:
How about `\S+`?

Reply:
Will that work for unicode too?

Reply:
I believe so, from my quick test it does. `\s*\S+.*`

Typically the validator isn't about stopping AMP creators from doing things they might not want to do. So I'm somewhat on the fence of whether this is helpful.

Reply:
I'm alright with either.